### PR TITLE
Missing link in the Orientation Guide #6143

### LINKED ISSUE #6143: Missing link in the Orientation Guide
Hello!

I was following along in the Orientation Guide when I came to the following point in the orientation task list:

> Phase B: Set up locally
>
> Set up TEAMMATES development environment on your computer.
> Important:
> Follow instructions to the letter. Install the specified versions of the tool stack, not the latest versions.
> If you encounter any problems, refer to the Developer Troubleshooting Guide.

The problem is that there is no link to the instructions referenced. I'm guessing there should have been a link to the settingUp doc,

### PR DESCRIPTION
**Pull request by wwSchrader**
Fixes #6143

Add link to setting up guide.

### CODE REVIEW

**Comment by wkurniawan07** [comment]
@wwSchrader thanks for your contribution!
As per our contributions guideline, a PR needs to address an issue (or more, although rarely the case). Can you open an issue which will be addressed by this PR, and modify this PR's description to include `Fixes #1111` (where `1111` is to be replaced with the issue number) and title to match the issue title, e.g `Do something #1111`?

File: docs/orientation-guide.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ _**Phase B**: Set up locally_
 
 1.  Set up TEAMMATES development environment on your computer.  
     Important:  
-    * Follow instructions to the letter. _Install the specified versions of the tool stack, not the latest versions._  
+    * [Follow instructions to the letter.](./settingUp.md) _Install the specified versions of the tool stack, not the latest versions._
```
Comment (wkurniawan07) [inline docs/orientation-guide.md]: This hyperlink is better at the main sentence, i.e. at "Set up TEAMMATES development environment". Also, you don't need the `./` prefix.

**Comment by wwSchrader** [comment]
My bad! I opened an issue and updated the PR with references to it.

File: docs/orientation-guide.md (hunk 1 above)
Comment (wwSchrader) [inline docs/orientation-guide.md]: Good point! I'll make the suggested changes.

**Pull request merged**